weblate/checks/markup.py

```diff
@@ -108,6 +108,11 @@ class BBCodeCheck(TargetCheck):
     check_id = "bbcode"
     name = gettext_lazy("BBCode markup")
     description = gettext_lazy("BBCode in translation does not match source")
+    default_disabled = True
+
+    def __init__(self):
+        super().__init__()
+        self.enable_string = "bbcode-text"
```

Comment:
The check should no loger skip processin when no bbcode is found (the "Any BBCode in source?" block below) so that it can also detect added bbcode markup.